tests/base.py

```diff
@@ -27,8 +27,10 @@ def find_unused_port():
 
 
 class BaseTest(unittest.TestCase):
+
```

Comment:
Why did you add empty line before doc string every class? This looks unusual and bit ugly.